cli/cmd/tap.go

```diff
@@ -65,7 +65,7 @@ func init() {
 	tapCmd.Flags().BoolP(configStructs.AllNamespacesTapName, "A", defaultTapConfig.AllNamespaces, "Tap all namespaces")
 	tapCmd.Flags().StringP(configStructs.KubeConfigPathTapName, "k", defaultTapConfig.KubeConfigPath, "Path to kube-config file")
 	tapCmd.Flags().StringArrayP(configStructs.PlainTextFilterRegexesTapName, "r", defaultTapConfig.PlainTextFilterRegexes, "List of regex expressions that are used to filter matching values from text/plain http bodies")
-	tapCmd.Flags().Bool(configStructs.HideHealthChecksTapName, defaultTapConfig.HideHealthChecks, "hides requests with kube-probe or prometheus user-agent headers")
+	tapCmd.Flags().StringArray(configStructs.IgnoredUserAgentsTapName, defaultTapConfig.HealthChecksUserAgentHeaders, "hides requests with kube-probe or prometheus user-agent headers")
```

Comment:
Need to update the description

Reply:
updated

Reply:
I ended up with removing this flag (this option will be available only with editing from the config file)